drizzlepac/hapsequencer.py

```diff
@@ -341,7 +352,7 @@ def create_catalog_products(total_obj_list, log_level, diagnostic_mode=False, ph
             # rate of cosmic-ray contamination for the total detection product
             reject_catalogs = total_product_catalogs.verify_crthresh(n1_exposure_time)
 
-            if not reject_catalogs:
+            if not reject_catalogs or diagnostic_mode:
```

Comment:
There will need to be a change in catalog_utils.py (write_catalog).  The routines needs to be notified about the reject_catalogs as the routines, with this change, will always write the .ecsv file and only write the .reg file in diagnostic mode.  There will be times when it is only desired to write the .reg file.  I think this should be a different ticket.

Reply:
Removed in 88a1980 so that this can be addressed more fully in a separate PR.